english/netflix/netflix.md

```diff
@@ -4,11 +4,11 @@ networks in the marshmallow.
 
 What do you do?
 
-Especially since house of cards season 3 just became available. 
```

Comment:
Unfortunately, pull requests that delete or modify lines are not eligible for automatic merging.  If you wish to have this pull request auto-merged, please update it to only add lines.  If you don't think you've modified any lines, remember that I consider changing the whitespace (e.g. deleting a space at the end of a line) a modification, even though it may be hard to tell the difference on GitHub.